File: Enlazar.Database/Service.cs
using Enlazar.Database.Utilities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Enlazar.Database
{
    public class Service
    {
        public string id { get; set; }
        public string longitud { get; set; }
        public string latitud { get; set; }
        public string recolectorId { get; set; }
        public string userId { get; set; }
        public string address { get; set; }
        public string date { get ; set; }
        public string time { get ; set; }
        public int envasesCarton { get; set; }
        public int envasesPlasticos { get; set; }
        public int envasesVidrio { get; set; }
        public StateServices estado { get; set; }


    }
}

File: Enlazar/app/src/main/java/ar/com/unlam/enlazar/model/Service.kt
package ar.com.unlam.enlazar.model

import java.io.Serializable
import java.util.*

data class Service (
        val address:String? = null,
        val id:String? = null,
        val latitud:String? = null,
        val longitud:String? = null,
        val envasesPlasticos:Int?? = null,
        val envasesVidrio:Int?? = null,
        val envasesCarton:Int?? = null,
        val date: String? = null,
        val time:String? = null,
        val userId:String? = null,
        val recolectorId:String? = null,
        val estado:Int? = null,

        ): Serializable